docs/content/en/blog/releases/v0.13.0.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Release v0.13.0"
+linkTitle: "Release v0.13.0"
+date: 2021-07-20
+description: >
+  Release v0.13.0
+---
+
+## Changelog since v0.12.1
+
+### Notable Changes
+* Do not mask the diff output of ConfigMap ([#2276](https://github.com/pipe-cd/pipe/pull/2276))
+* Fix wrong plan-preview result in case the base branch was updated after PR is created ([#2273](https://github.com/pipe-cd/pipe/pull/2273))
```

Comment:
```suggestion
* Fix wrong plan-preview result in case the base branch was updated after PR had created ([#2273](https://github.com/pipe-cd/pipe/pull/2273))
```
nit

Reply:
👍